lib/synsbasen_api/resources/inspection_test_center.rb

```diff
@@ -12,6 +12,21 @@ class << self
       def all
         get("/v1/inspection_test_centers")
       end
+
+      # Performs a search for inspection test centers based on the provided criteria.
+      #
+      # @param args [Hash] Additional parameters to customize the search.
+      # @option args [String] :method The search method. Default is 'SELECT'.
+      # @return [ApiResponse] An instance of `ApiResponse` containing search results.
+      def search(args = {}, expand: [])
+        post(
+          "/v1/inspection_test_centers/search",
+          body: {
+            method: 'SELECT',
+          }.merge(args),
+          expand: expand
+        )
+      end
```

Comment:
Should almost be a setting on each of the models which supports search. I imagine you have this exact code in vehicle and inspection too.